FGEGraphics/ClientSystem/ShaderLocations.cs

```diff
@@ -42,4 +48,300 @@ public static class Common2D
         /// <summary>The rotation effect to apply.</summary>
         public const int ROTATION = 4;
     }
+
+    /// ===== FORWARD RENDERING SHADERS =====
```

Comment:
... What.

Reply:
Well it was this or putting them in a forward/deferred subclass meaning I'd have to rework all of the deferred locations (if there is a way I can just do that part automatically that'd be great to know)

Reply:
But in the summaries, it's clear what they belong to.

Reply:
Yes it should be in a subclass

Reply:
re automation of it, just, uh, find-and-replace-in-files? VS Code has a really clean interface to do that quick, though VS can too